sonoptix_driver:
  ros__parameters:
    ip_address: "192.168.1.211"
    range: 15
    gain: 4
    angle_range: 2.09
    intensity_threshold: 127